azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -131,6 +132,7 @@ def __init__(self):
             self._cgroups_api = None
             self._agent_cpu_cgroup_path = None
             self._agent_memory_cgroup_path = None
+            self._lock = threading.RLock()
```

Comment:
can we rename this to help document what the lock is meant to be used for?

_check_cgroups_lock?

Reply:
updated.